<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminMessages</class>
 <widget class="QWidget" name="adminMessages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1152</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>20</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(178, 8, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pictures.qrc">:/logo/filename.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/AjaxLogo/filename.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>230</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(41, 39, 40);
border-color: rgb(41, 39, 40);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>100</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>160</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>220</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>280</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>1</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/profile.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/message.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/add-user.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/upload-mail.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Hi Dear Shantia</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Messges</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Add People</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Send Assertion</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="classIcon.qrc">:/classIcon/icons8-class-50.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>340</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Add CLass</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>41</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(178, 8, 55);
border-color: rgb(178, 8, 55);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>2</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="icon.qrc">:/icon/information.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string>Class Information</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>400</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Montserrat&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>frame_5</zorder>
   <zorder>frame_6</zorder>
   <zorder>frame_7</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_8</zorder>
   <zorder>frame_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>frame_15</zorder>
   <zorder>label_21</zorder>
   <zorder>pushButton_7</zorder>
   <zorder>frame_9</zorder>
   <zorder>label_15</zorder>
   <zorder>pushButton_6</zorder>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>41</width>
     <height>46</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>font: 9pt &quot;Montserrat&quot;;
color: rgb(178, 8, 55);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="arrowIcon.qrc">:/new/prefix1/icons8-left-arrow-50.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="backToMenu">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>791</width>
     <height>441</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>789</width>
      <height>439</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>label</zorder>
  <zorder>frame_2</zorder>
  <zorder>label_9</zorder>
  <zorder>label_14</zorder>
  <zorder>backToMenu</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <resources>
  <include location="classIcon.qrc"/>
  <include location="icon.qrc"/>
  <include location="arrowIcon.qrc"/>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
